templates/pricing/devices.html

```diff
@@ -0,0 +1,649 @@
+
+    <div class="col-6 p-divider__block">
+      <h3 class="p-heading--five">Ongoing testing and security updates</h3>
+      
+      <p>For an annual charge of $25k, Canonical will perform continuous security maintenance and testing on your board. When a security issue is fixed in Ubuntu, the fix will be tested on your board. These security updates will be published to the snap store for use on your device. Update Control can be used to require additional testing and validation by your team before these updates are automatically applied to your devices. Testing will cover kernel and base system debs and snaps, along with any board-specific and app-specific tests you specify.</p>
+    </div>
+  </div>
+</section>
+
+<section class="p-strip--light">
+  <div class="row u-vertically-center">
+    <div class="col-8">
+      <h2>Certification of boards and devices</h2>
+      <p class="p-heading--four">Guarantee long-term security, application compatibility, support and operational consistency with the Ubuntu Certified logo for enterprise-grade devices.</p>
+      <p>Long-term automated Canonical testing of every update and security patch ensures devices perform reliably over the full lifetime of the certified Ubuntu LTS release. Certified devices must run a Canonical kernel. We require four units of every certified board or device for continuous testing. In the case of devices built with existing certified boards, running their standard certified kernel, we reduce this requirement to two devices.</p>
+
+      <p>Certification for devices that include pre-installed snaps can include validation of the integrity of those snaps.</p>
+    </div>
+
+    <div class="col-2 col-start-large-10"><img src="https://assets.ubuntu.com/v1/706b259a-ubuntu_certified_hex.svg" alt="Ubuntu Certified"></div>
+  </div>
+</section>
+
+<section class="p-strip">
+  <div class="u-fixed-width">
+    <h2>Device support for manufacturers</h2>
+    <h3 class="p-heading--four">Access to Canonical engineering for device engineering and development teams.</h3>
+    <p>Level 4 (L4) support for device manufacturers enables access to Canonical engineering and techops teams to help troubleshoot and resolve issues in Ubuntu and Ubuntu Core. Covers kernel, device integration, base OS, snap security and update mechanisms, networking, communications and graphics.</p>
+
+    <div style="overflow-x: auto;">
+      <table class="p-table">
+        <thead>
+          <th>Embedded Ubuntu Support for Manufacturers</th>
+          <th class="u-align--center">Standard</th>
+          <th class="u-align--center">Advanced</th>
+        </thead>
+        <tbody>
+          <tr>
+            <td>Physical server</td>
+            <td class="u-align--center">$750</td>
+            <td class="u-align--center">$1,500</td>
+          </tr>
+          <tr style="background-color: #e5e5e5">
+            <td>Phone and ticket support</td>
+            <td class="u-align--center">Monday - Friday</td>
```

Comment:
on https://ubuntu.com/legal/ubuntu-advantage-service-description we changed Monday-Friday to 24/5 we should do the same here and update the copy doc